docker-compose.yml

```diff
@@ -83,6 +83,12 @@ services:
     <<: *txs-worker
     command: docker/web/celery/scheduler/run.sh
 
+  txs-flower:
```

Comment:
I would add this in a separate docker-compose file: https://docs.docker.com/compose/extends/